lib/extended_json.js

```diff
@@ -265,38 +265,68 @@ function serializeValue(value, options) {
   return value;
 }
 
+const bsonTypeMappings = {
```

Comment:
nit: since this is a constant can we use caps (`BSON_TYPE_MAPPINGS`)

Reply:
OK done.